.eslintrc.json

```diff
@@ -43,36 +43,51 @@
     },
     { "files": ["**/__tests__/**"], "env": { "jest": true } },
     {
+      "extends": [
+        "plugin:@typescript-eslint/recommended",
+        "plugin:@typescript-eslint/stylistic"
+      ],
       "files": ["**/*.ts", "**/*.tsx"],
       "parser": "@typescript-eslint/parser",
       "parserOptions": {
-        "ecmaVersion": 2018,
```

Comment:
Unnecessary configuration line. Per https://typescript-eslint.io/packages/parser#ecmaversion, 2018 is the default now.